plugins/lookup/github_app_access_token.py

```diff
@@ -66,13 +68,25 @@
   elements: str
 """
 
-
 try:
-    from jwt import JWT, jwk_from_pem
+    import jwt
     HAS_JWT = True
 except ImportError:
     HAS_JWT = False
 
+HAS_PYTHON_JWT = False  # vs pyjwt
+if HAS_JWT and hasattr(jwt, 'JWT'):
+    HAS_PYTHON_JWT = True
+    from jwt import jwk_from_pem, jwt_instance
+
+if not HAS_PYTHON_JWT:
+    try:
+        from cryptography.hazmat.primitives import serialization
+        HAS_CRYPTOGRAPHY = True
+    except ImportError:
+        HAS_CRYPTOGRAPHY = False
```

Comment:
I'd leave this `if` away:
```suggestion
try:
    from cryptography.hazmat.primitives import serialization
    HAS_CRYPTOGRAPHY = True
except ImportError:
    HAS_CRYPTOGRAPHY = False
```